index.js

```diff
@@ -0,0 +1,6 @@
+/* jshint node: true */
+'use strict';
+
+module.exports = {
+  name: 'model-fragments'
```

Comment:
@workmanw good call on needing to check `ember install` I haven't actually tried that. As for the namespace I'm in favor of keeping it the same for backwards compatibility. We could definitely revisit it in the future though.